release-notes/opensearch.release-notes-1.3.10.md

```diff
@@ -0,0 +1,11 @@
+## 2023-05-18 Version 1.3.10 Release Notes
+
+### Upgrades
+- Upgrade Netty from 4.1.90.Final to 4.1.91.Final , ASM 9.4 to ASM 9.5, ByteBuddy 1.14.2 to 1.14.3 ([#6981](https://github.com/opensearch-project/OpenSearch/pull/6981))
+- Upgrade `org.gradle.test-retry` from 1.5.1 to 1.5.2
+- Upgrade `org.apache.hadoop:hadoop-minicluster` from 3.3.4 to 3.3.5
```

Comment:
[Snakeyaml](https://github.com/opensearch-project/OpenSearch/blob/1.x/CHANGELOG.md?plain=1#L11) entry is missing here